leshan-server-core/src/main/java/org/eclipse/leshan/server/queue/impl/QueuedRequestFactoryImpl.java

```diff
@@ -64,10 +67,20 @@ public long getRequestId() {
             return requestId;
         }
 
+        /*
+         * (non-Javadoc)
+         * 
+         * @see org.eclipse.leshan.server.queue.QueuedRequest#getRequestTicket()
+         */
+        @Override
+        public String getRequestTicket() {
+            return this.requestTicket;
+        }
+
         @Override
         public String toString() {
-            return "QueuedRequestImpl [downlinkRequest=" + downlinkRequest + ", endpoint=" + endpoint + ", requestId="
-                    + requestId + "]";
+            return "QueuedRequestImpl [requestTicket=" + requestTicket + ", downlinkRequest=" + downlinkRequest
```

Comment:
In the leshan we try to use String.format.
See [Leshan Code rules](https://github.com/eclipse/leshan/wiki/Code-&-design-guidelines).
_"toString should be implemented using String format"_